src/apps/leaderboards/models.py

```diff
@@ -55,6 +55,8 @@ def compute(self, scores):
 class SubmissionScore(models.Model):
     column = models.ForeignKey(Column, on_delete=models.CASCADE)
     score = models.DecimalField(max_digits=20, decimal_places=10)
+    # task_pk to specify which task run the submission
+    task_pk = models.PositiveIntegerField(null=True, blank=True)
```

Comment:
the column name should just be `task`, I don't think we need the comment as well

Reply:
And this should point to a `Task` instead of just the integer?

Reply:
Sorry for spam -- just noticed @dde6khkg made it into `task` here it looks like:
https://github.com/codalab/competitions-v2/pull/463/

Reply:
hello eric, when i am trying to rename column `task_pk` to `task`, it comes an error of relationship of `leaderboard_submissionscore` already exists.

Reply:
sorry, after read link you provided, i still not get it why it should point to a Task instead of just the integer here? 

integer here is used to store task_pk send by compute_worker in order to distinguish them independently.
I don't know if I'm making myself clear,  feel free to ask me if you have many questions, many thanks!!!

Reply:
Why track that integer instead of just doing a foreign key?

Reply:
got it, Thanks for your patience, I've modified it to the foreign key way, the only thing I need to do is I need to query out the task object! 

Can you review the code again? ty!!!

Reply:
hello @ckcollab , any comment on this PR?

Reply:
please wait for the other PRs to be merged which will change how we handle this one, ty for patience!! so close to merging the other ones.. fixing up tests and such